packages/core/src/external/commonwell/cq-bridge/coverage-enhancer-local.ts

```diff
@@ -69,7 +69,7 @@ export class CoverageEnhancerLocal extends CoverageEnhancer {
             log,
           });
         } catch (error) {
-          const msg = `ERROR at org chunk ${i} (relative) / ${i + fromOrgChunkPos} (absolute)`;
+          const msg = `ERROR at org chunk ${i} (relative) / ${chunks.length - 1} (absolute)`;
```

Comment:
Sorry for the confusion (i think I wrote this), but the idea here was not to indicate X of Y (progress), but X and Y (two diff information); like, not trying to indicate the chunk number of the total number of chunks, but indicating the chunk number in relative and absolute terms:
- if it fails, this indicated which chunk failed;
- "relative" indicates the chunk in relation to the number of chunks being executed at this run (starting at zero);
- "absolute" indicates the chunk in relation to the total number of chunks (starting at `fromOrgChunkPos`).

Maybe we can just change the syntax so it doesn't give the idea of progress (X of Y)?